apps/ledger-live-desktop/static/i18n/en/app.json

```
@@ -6374,6 +6374,16 @@
     },
     "PinNotSet": {
       "title": "PIN not set"
+    },
+    "TransactionBroadcastError": {
+      "title": "Transaction Broadcast Unsuccesful",
+      "description": "Your transaction failed to broadcast on the {{networkName}} network. Your {{currencyName}} have not be transferred and remain in your account.",
```

I realized writting the prev comment that there's a typo in Figma !
```suggestion
      "description": "Your transaction failed to broadcast on the {{networkName}} network. Your {{currencyName}} have not been transferred and remain in your account.",
```